libraries/botbuilder-core/src/turnContext.ts

```diff
@@ -320,7 +320,11 @@ export class TurnContext {
      * @param activity New replacement activity. The activity should already have it's ID information populated.
      */
     public updateActivity(activity: Partial<Activity>): Promise<void> {
-        return this.emit(this._onUpdateActivity, activity, () => this.adapter.updateActivity(this, activity));
+
+        let conversationReference = TurnContext.getConversationReference(this._activity);
```

Comment:
@Virtual-Josh and @EricDahlvang this change is the same as https://github.com/microsoft/botbuilder-js/pull/1169, correct? 

... It's a race! 

@EricDahlvang can you change the `let` to `const` in this method?

Reply:
We took #1169 ... rebasing this PR with those changes (Kyle's implementation is cleaner :) )